Index: git/base/framework/python/setup.py
===================================================================
--- git.orig/base/framework/python/setup.py
+++ git/base/framework/python/setup.py
@@ -55,8 +55,16 @@ setup(name='scapy',
                 'sca/bulkio/BULKIO',
                 'sca/bulkio/BULKIO__POA',
                 'sca/cf',
+                'sca/cf/BULKIO',
+                'sca/cf/BULKIO__POA',
                 'sca/cf/CF',
                 'sca/cf/CF__POA',
+                'sca/cf/Extended_CF',
+                'sca/cf/Extended_CF__POA',
+                'sca/cf/ExtendedCF',
+                'sca/cf/ExtendedCF__POA',
+                'sca/cf/PortTypes',
+                'sca/cf/PortTypes__POA',
                 'sca/cf/StandardEvent',
                 'sca/cf/StandardEvent__POA',
                 'sca/events',
